openapi/openapi.yaml

```diff
@@ -12,6 +12,9 @@
 'servers':
 - 'url': '/control'
 
+'security':
+  - 'basicAuth': []
```

Comment:
There shouldn't be an indentation here, according to our [*YAML* style].  Other than that, it's good to go.

[*YAML* style]: https://github.com/AdguardTeam/AdGuardHome/blob/master/HACKING.md#yaml

Reply:
fixed